File: MicroservicesProject/src/UserManagementService/springboot-app/src/main/java/com/UserManagementService/springbootapp/Responses/LoginResponse.java
package com.UserManagementService.springbootapp.Responses;

public class LoginResponse {
    public String Sender;
    public String ResponseType;
    public String Username;
    public boolean LoggedIn;
    public String Message;

    public LoginResponse(String sender, String responseType, String username, boolean loggedIn, String message) {
        Sender = sender;
        ResponseType = responseType;
        Username = username;
        LoggedIn = loggedIn;
        Message = message;
    }
    public LoginResponse(String sender, String responseType, boolean loggedIn, String message) {
        Sender = sender;
        ResponseType = responseType;
        LoggedIn = loggedIn;
        Message = message;
    }

}

File: MicroservicesProject/src/WPF_Client/ClientBackLogic/Responses/LoginResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClientBackLogic.Responses
{
    public class LoginResponse: BaseResponse
    {
        public string Username { get; set; }
        public bool LoggedIn { get; set; }
    }
}
